### PR TITLE
feat: menu on sidebar (part 2)

### PR DESCRIPTION
**Pull request by preschian**
**Thank you for your contribution** to the [KodaDot - One Stop Shop for Polkadot NFTs](https://kodadot.xyz).

👇 __ Let's make a quick check before the contribution.

## PR Type

- [ ] Bugfix
- [x] Feature
- [ ] Refactoring

## Context

- [x] Related #5820
- [x] menus on sidebar
- [x] adjust menu + mobile version
- [x] set identity banner

<details><summary>Screenshot</summary>
<p>

localhost_9090__redesign=true (1)

</p>
</details> 

#### Had issue bounty label?

- [x] My DOT address: [Payout](https://canary.kodadot.xyz/dot/transfer/?target=1xjvRADwdJcnmUCLWerEHR4iGCT5EBTm4D4fzLLg4LcAC9p)

### <samp>🤖 Generated by Copilot at f6e4bbf</samp>

This pull request introduces several improvements and fixes to the UI and functionality of the connect wallet modal and the multiple balances component. It also adds support for the Basilisk network and fixes some code style and syntax issues in the SCSS files. The main files affected are `components/balance/MultipleBalances.vue`, `components/common/ConnectWallet/WalletAsset.vue`, `composables/usePrefix.ts`, `styles/abstracts/_theme.scss`, and `stores/lang.ts`.

### <samp>🤖 Generated by Copilot at f6e4bbf</samp>

> _Sing, O Muse, of the glorious deeds of the code reviewers,_
> _who with skill and wisdom examined the pull request,_
> _and found many improvements and fixes in the wallet components,_
> _that display the user's assets and options, like the gods on Olympus._

### CODE REVIEW

**Comment by exezbcz** [comment]
Is there any deploy preview, or is it waiting for part 1 to be merged?

**Comment by preschian** [comment]
> Is there any deploy preview, or is it waiting for part 1 to be merged?

oh, no need to wait for part 1. let me change the target branch first

**Comment by exezbcz** [comment]
woho, getting there 

### feedback
image
- the settings dissappear or changes its default color when clicked 

image

- the portfolio view should redirect to asset page

- the sidebar should close when you click on the menu or you get redirected

- add scroll when the asset preview is too long 
image
   
   - top part with profile and cross stays fixed, the rest scrolls 
   - or maybe make the first section scroll
image
 

- please center out the dropdown with the link-button 
image

**Comment by preschian** [comment]
> * the portfolio view should redirect to asset page

@exezbcz did you mean `/assets` page? seems like only `Basilisk` chain has that page

**Comment by roiLeo** [comment]
> > * the portfolio view should redirect to asset page
> 
> @exezbcz did you mean `/assets` page? seems like only `Basilisk` chain has that page

I think `/assets` page will be replaced by #5759 (#4119 design)

You could try to show it only on bsx/snek chain

**Comment by preschian** [comment]
> I think `/assets` page will be replaced by #5759 (#4119 design)
> 
> You could try to show it only on bsx/snek chain

aahh, i see, updated

File: components/common/ConnectWallet/WalletAssetMenu.vue
Comment (roiLeo) [inline components/common/ConnectWallet/WalletAssetMenu.vue:28]:
```suggestion
          mobile-modal
```

File: stores/identity.ts
Comment (roiLeo) [inline stores/identity.ts:156]: why remove?

File: styles/global.scss
Comment (roiLeo) [inline styles/global.scss:260]: hmmm looks weird but ok

**Review by roiLeo — APPROVED** [review]
When will you remove `?redesign=true`?

any way to use `NeoDropdown` instead of `b-dropdown`?

otherwise code lgtm

File: stores/identity.ts
Comment (preschian) [reply stores/identity.ts:156]: `setAuth` is also used when changing the prefix/chain. for multi balances, no need to reset that. because the address and the balance are already stored in multi chain

File: styles/global.scss
Comment (roiLeo) [reply styles/global.scss:260]: Imo this can have side effects, maybe writte it within `.connect-wallet-modal` class (scoped component class)

**Review by roiLeo — COMMENTED** [review]
Sometimes Identity isn't loading

Capture d’écran 2023-06-06 à 9 57 14 AM

File: styles/global.scss
Comment (preschian) [reply styles/global.scss:260]: I found out that this applies everywhere. I put it here just in case we also have that class in a different place. wdyt?

for example, you can reproduce that by adding `has-text-gray` class to our footer link

**Comment by preschian** [comment]
> Sometimes Identity isn't loading

oh, we don't have a cache for identity anymore. so the identity will call it directly to the WS endpoint. so it will depend on WS endpoint can resolve it or not. could you check, if is there any error on the console or WS endpoint?

or, we can improve it by store identity to pinia stores? wdyt?
so, not only call balances from cross-chain but check the identity also and store to pinia

**Comment by roiLeo** [comment]
> oh, we don't have a cache for identity anymore. so the identity will call it directly to the WS endpoint. so it will depend on WS endpoint can resolve it or not. could you check, if is there any error on the console or WS endpoint?

maybe ws endpoint not connected? see screenshot

Capture d’écran 2023-06-06 à 10 18 17 AM

> or, we can improve it by store identity to pinia stores? wdyt? so, not only call balances from cross-chain but check the identity also and store to pinia

I think it was already the case (Identity Vuex store => pinia) but yes this would be perfect! so we'll be able to reuse Identity valu between components and not call again ws endpoint if identiy is already fetched

File: styles/global.scss
Comment (roiLeo) [reply styles/global.scss:260]: I though you changed line 254, alright let it roll like this

**Comment by preschian** [comment]
> maybe ws endpoint not connected? see screenshot

oohh, seems like your identity on Basilisk is still empty https://basilisk.subscan.io/account/bXhWeVLtArWauM8UmbaVdDmXmo5uSDk6DAa2YxQ5zbt2sDcbW

compared to this https://basilisk.subscan.io/account/bXgcVtxi8G4AHk2MXcqyKv5W5eEyPnyoEggw2zoHTXBDbHMnm

**Comment by roiLeo** [comment]
> > maybe ws endpoint not connected? see screenshot
> 
> oohh, seems like your identity on Basilisk is still empty https://basilisk.subscan.io/account/bXhWeVLtArWauM8UmbaVdDmXmo5uSDk6DAa2YxQ5zbt2sDcbW


Oh, shoot! sorry for that I forgot that it was set on snek chain

Capture d’écran 2023-06-06 à 10 48 03 AM

**Comment by prury** [comment]
Incoming offers redirecting to the wrong address on BSX (incoming-offers instead of incomingoffers)

**Comment by yangwao** [comment]
@exezbcz  👀

**Comment by preschian** [comment]
need 1 more approval, so the PR can be merged

Screenshot 2023-06-09 at 3 01 40 PM

File: composables/usePrefix.ts
Comment (vikiival) [inline composables/usePrefix.ts:46]: I would be better to have a `isChain` composable

**Pull request merged**
